buildenv

```diff
@@ -1,84 +1,195 @@
 export ZOPEN_STABLE_URL="https://github.com/boostorg/boost.git"
-#export ZOPEN_STABLE_TAG="boost-1.82.0"
-export ZOPEN_DEV_BRANCH="boost-1.82.0"
+export ZOPEN_DEV_TAG="boost-1.82.0"
 export ZOPEN_STABLE_DEPS="cmake make zoslib which coreutils comp_clang sed jq"
 export ZOPEN_DEV_URL="https://github.com/boostorg/boost.git"
-export ZOPEN_DEV_DEPS="cmake make zoslib which coreutils comp_clang sed jq"
+export ZOPEN_DEV_DEPS="cmake make zoslib which coreutils comp_clang sed jq git findutils diffutils bzip2 zlib"
 export ZOPEN_BUILD_LINE="DEV"
 export ZOPEN_CLONE_SUBMODULES="yes"
 export ZOPEN_COMP="clang"
 export ZOPEN_CXXFLAGS ="-m64"
```

Comment:
```suggestion
export ZOPEN_CXXFLAGS ="-m64"
```
There's a space before the =. Also, is this needed since comp_clang port supplies the CXXFLAGS?

Reply:
Right. This isn't needed. I have removed it.